modules/serverless-monitoring-services-configuration-scraping.adoc

```diff
@@ -0,0 +1,68 @@
+[id="serverless-monitoring-services-configuration-scraping_{context}"]
+= Configuration for scraping custom metrics
+
+The scraping of custom metrics is performed by an instance of Prometheus purposed for user workload monitoring. Once you enable user workload monitoring and create the application, you need a configuration that defines how the monitoring stack will scrape the metrics.
+
+The following sample configuration defines the ksvc for your application and configures the service monitor. The exact configuration depends on your application and how it exports the metrics.
+
+[source,yaml]
+----
+apiVersion: serving.knative.dev/v1 <1>
+kind: Service
+metadata:
+  name: helloworld-go
+spec:
+  template:
+    metadata:
+      labels:
+        app: helloworld-go
+      annotations:
+    spec:
+      containers:
+      - image: docker.io/skonto/helloworld-go:metrics
```

Comment:
minor nit:   should we use `quay.io`, as that is a RHT prodct / serivce ?

We could create a new repo inside of `quay.io/openshift-knative` for the image

Reply:
@matzew That would be best, thanks.